recipe/activate.sh

```diff
@@ -32,6 +33,7 @@ then
     #
     export CXXFLAGS="${CXXFLAGS} -DBOOST_MATH_DISABLE_FLOAT128"
     export LDFLAGS="${LDFLAGS}"
+    export LDFLAGS="${LDFLAGS} -Wl,-rpath,$PREFIX/lib" 
```

Comment:
On Linux, this need to be `=` instead of `,`. IOW `-Wl,-rpath=$PREFIX/lib`.

Reply:
Both are correct in linux right?

Reply:
Hmm...I don't think so, but it has been a while since I checked.

Reply:
I've been using it on many recipes without any problems.

Reply:
Do you have an example?

Reply:
https://github.com/conda-forge/gmp-feedstock/blob/2d45981ccd505a8049bedd74e91ea58ac16058b3/recipe/run_test.sh

Reply:
Not seeing any warnings in the CI log. So guess it works fine.